Parent_read_points.py

```diff
@@ -0,0 +1,61 @@
+
+def read_file(filename):
+    """
+    Read and return the contents of a file.
+
+    Para:
+        filename (str): The name of the file to be read.
+
+    Returns:
+        str: The contents of the file.
+    """
+    with open(filename, "r") as f:
+        return f.read()
+
+def read_exam_data(exam_names, student_number):
+    """
+    Read exam data from files for a given list of exam names and student number.
+
+    Para:
+        exam_names (list): A list of exam names.
+        student_number (int): 1 for Student1, 2 for Student2.
+
+    Returns:
+        dict: A dictionary where keys are exam names and values are exam data.
+    """
+    exam_data = {}
+    for exam_name in exam_names:
+        data = read_file(f"{'max' if student_number == 1 else 'john'}_{exam_name}_saved_points.txt")
```

Comment:
Looks an instance of student would probaby know the given name of the student. Keeping the above comment in mind, how could the be structured differently.?